xercesc.sh

```diff
@@ -11,6 +11,8 @@ prefer_system_check: |
 prepend_path:
   ROOT_INCLUDE_PATH: "$XERCESC_ROOT/include"
   LD_LIBRARY_PATH: "$XERCESC_ROOT/lib"
+env:
```

Comment:
I think this would need to be prepend_path:

```suggestion
prepend_path:
```

because otherwise Xerces will overwrite everything else.

Reply:
It should be in `prepend_path` to handle the case where `CMAKE_PREFIX_PATH` is empty in the outside environment.

There's already a `prepend_path` block above, so merge it into that one (as `CMAKE_PREFIX_PATH: "$XERCESC_ROOT"`) to avoid wiping out the statements already there. Incidentally, aliBuild should already handle `LD_LIBRARY_PATH`, so we probably don't need that one...